server/lib/passport.js

```diff
@@ -195,18 +195,16 @@ if (LdapStrategy) {
 
         } catch (err) {
             if (err instanceof interoperableErrors.NotFoundError) {
-                const userId = await users.create(null, {
+                const userId = await users.create(contextHelpers.getAdminContext(), {
                     username: profile[config.ldap.uidTag],
-                    email: profile.mail,
                     role: config.ldap.newUserRole,
-                    namespace: config.ldap.newUserNamespaceId,
-                    name: profile.cn
+                    namespace: config.ldap.newUserNamespaceId
                 });
 
                 return {
                     id: userId,
-                    username: profile.uid,
-                    name: profile.cn,
+                    username: profile[config.ldap.uidTag],
+                    name: profile[config.ldap.nameTag],
                     email: profile.mail,
```

Comment:
Please, could you also replace this by the tag from config. This was forgotten in one of the previous commits. And the same on line 192. Thanks.